i2c/sensirion_i2c.h

```diff
@@ -165,6 +166,146 @@ int16_t sensirion_i2c_delayed_read_cmd(uint8_t address, uint16_t cmd,
 int16_t sensirion_i2c_read_cmd(uint8_t address, uint16_t cmd,
                                uint16_t* data_words, uint16_t num_words);
 
+/**
+ * sensirion_i2c_add_command_to_buffer() - Add a command to the buffer at
+ *                                         offset. Adds 2 bytes to the buffer.
+ *
+ * @param offset  Offset of the next free byte in the buffer.
```

Comment:
in the docstr, `offset` and `buffer` params are switched wrt. the argument order in the function (check all functions)